iOS-TvShows/iOS-TvShows/Common/UI/Bundles/BundleFont.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  BundleFont.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 09.11.2021..
+//
+
+enum BundleFont: String, CaseIterable {
+
+    case regular = "SFProDisplay-Regular"
```

Comment:
Hmm why did you add these fonts? Those are the `UIFont.system`, no need for custom adding them 😄

Reply:
Didnt know :D